pom.xml

```diff
@@ -25,6 +25,7 @@
     <cmake.generate.extra></cmake.generate.extra>
     <cmake.generate.skip>false</cmake.generate.skip>
     <javah.skip>false</javah.skip>
+    <skip.fat.jar>true</skip.fat.jar>
```

Comment:
Just a thought.... `with-dependencies` may be a more intuitive name?

Reply:
Good idea. While "fat jar" seems to be more common in the maven commit, with-dependencies is the official term and not only sounds more proper, but also gives a hint of what is included.

Reply:
Done